### PR TITLE
Correct spelling of database

### PR DESCRIPTION
**Pull request by fthrslntgy**
Database is "database", not "data base" 😂 
##### Checklist
- [X] documentation is added or updated
- [X] tests are added or updated

credit: @yacanka

### CODE REVIEW

File: apps/ca.c [hunk 1]
```diff
@@ -1118,7 +1118,7 @@ int ca_main(int argc, char **argv)
             if (!rotate_index(dbfile, "new", "old"))
                 goto end;
 
-            BIO_printf(bio_err, "Data Base Updated\n");
+            BIO_printf(bio_err, "Database Updated\n");
```
Comment (tom-cosgrove-arm) [inline apps/ca.c:1121]: I think if we're going to update "Data Base" to "Database", we should also update "Updated" to "updated".

These aren't titles, they are just notices to say what the program has done, so normal sentence case is appropriate.

File: apps/ca.c [hunk 2]
```diff
@@ -1297,7 +1297,7 @@ int ca_main(int argc, char **argv)
...
```
Comment (tom-cosgrove-arm) [inline apps/ca.c:1300]: I think if we're going to update "Data Base" to "Database", we should also update "Updated" to "updated".

These aren't titles, they are just notices to say what the program has done, so normal sentence case is appropriate.

File: apps/ca.c (hunk 1 above)
Comment (fthrslntgy) [reply apps/ca.c]: Done.

File: apps/ca.c (hunk 2 above)
Comment (fthrslntgy) [reply apps/ca.c]: Done.

**Comment by fthrslntgy** [comment]
> Please amend your commit message to contain a line "CLA: trivial".

Trivial line added to commit message.

**Review by paulidale — APPROVED** [review]
agreed trivial

**Review by tom-cosgrove-arm — APPROVED** [review]

**Comment by tom-cosgrove-arm** [comment]
agreed trivial

**Review by t8m — APPROVED** [review]
ok with trivial

**Review by raghu-007 — APPROVED** [review]

**Comment by tmshort** [comment]
Squashed to a single commit and merged to `master`, `openssl-3.1` and `openssl-3.0`. There was a trivial conflict going into 3.1 that I resolved. (For some reason, one of the context lines in 3.1 /3.0 has an extra semi-colon).

Thank you for your contribution!

**Pull request closed without merging**